events/bingo/bingoCardApp/bingoCard_app.js

```diff
@@ -319,29 +321,37 @@
         }
     }
 
-    /* ON APP START: Setup app UI, button, and messaging between it's html page and this script */
+    // When the script starts up, setup AppUI and call `cacheSounds()`.
+    // Also hook up necessary signals and open the app's UI.
     var ui;
     var AppUi = Script.require('appUi');
-    var appPage = Script.resolvePath('bingoCard_ui.html?12');
+    var appPage = Script.resolvePath('ui/bingoCard_ui.html?13');
     function startup() {
         ui = new AppUi({
             buttonName: "BINGO",
             home: appPage,
             graphicsDirectory: Script.resolvePath("assets/icons/"),
             onOpened: onOpened,
             onClosed: onClosed,
-            onMessage: onWebEventReceived
+            onMessage: onWebEventReceived,
+            sortOrder: 1
         });
         cacheSounds();
+        AvatarList.avatarSessionChangedEvent.connect(sessionChanged);
```

Comment:
What are the cases this happens?   What I was mainly wondering is does the avatarAddedl(not sure the exact name)  to the domain signal also fire when this happens?

Reply:
i thought avataradded was for all avatars being added to your avatarlist?

this signal is fired when you get a new sessionuuid, which happens when the user's domain changes.

Reply:
I was looking at the code, but I still couldn't figure if it was for everyone, or just your avatar. 

what I was looking at:
```AvatarHashMap::AvatarHashMap() {
    auto nodeList = DependencyManager::get<NodeList>();

    auto& packetReceiver = nodeList->getPacketReceiver();
    packetReceiver.registerListener(PacketType::BulkAvatarData, this, "processAvatarDataPacket");
    packetReceiver.registerListener(PacketType::KillAvatar, this, "processKillAvatar");
    packetReceiver.registerListener(PacketType::AvatarIdentity, this, "processAvatarIdentityPacket");
    packetReceiver.registerListener(PacketType::BulkAvatarTraits, this, "processBulkAvatarTraits");

    connect(nodeList.data(), &NodeList::uuidChanged, this, &AvatarHashMap::sessionUUIDChanged);

    connect(nodeList.data(), &NodeList::nodeKilled, this, [this](SharedNodePointer killedNode){
        if (killedNode->getType() == NodeType::AvatarMixer) {
            clearOtherAvatars();
        }
    });
}
```

Reply:
it's not totally clear to me either from inspection of the code. from in-world behavior it seems to only be for your avatar.